examples/step-27/doc/intro.dox

```diff
@@ -645,36 +645,17 @@ memory for all computations.
 
 A solution to the problem has already been discussed in step-11
 and step-18. It used an intermediate object of type
-CompressedSparsityPattern. This class uses a different memory storage scheme
+DynamicSparsityPattern. This class uses a different memory storage scheme
 that is optimized to <i>creating</i> a sparsity pattern when maximal numbers
 of entries per row are not accurately available, but is unsuitable for use as
 the sparsity pattern actually underlying a sparse matrix. After building the
 intermediate object, it is therefore copied into a true SparsityPattern
 object, something that can be done very efficient and without having to
 over-allocate memory. Typical code doing this is shown in the documentation of
-the CompressedSparsityPattern class. This solution is slower than directly
+the DynamicSparsityPattern class. This solution is slower than directly
 building a SparsityPattern object, but only uses as much memory as is really
 necessary.
 
-As it now turns out, the storage format used in the
```

Comment:
Ah, a German wrote that first half sentence. @drwells probably has this on his list ;-)